api/transport_windows.go

```diff
@@ -0,0 +1,17 @@
+package api
```

Comment:
Good point! We could just use it for all platforms. I'll change this back to just use npipe.